crates/indexer/src/models/token_models/token_claims.rs

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use super::{
+    token_utils::TokenWriteSet,
+    tokens::{TableHandleToOwner, TableMetadataForToken},
+};
+use crate::schema::current_token_pending_claims;
+use aptos_api_types::{DeleteTableItem as APIDeleteTableItem, WriteTableItem as APIWriteTableItem};
+use bigdecimal::{BigDecimal, Zero};
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, FieldCount, Identifiable, Insertable, Queryable, Serialize)]
+#[diesel(primary_key(token_data_id_hash, property_version, from_address, to_address))]
+#[diesel(table_name = current_token_pending_claims)]
+pub struct CurrentTokenPendingClaim {
+    pub token_data_id_hash: String,
+    pub property_version: BigDecimal,
+    pub from_address: String,
+    pub to_address: String,
+    pub collection_data_id_hash: String,
+    pub creator_address: String,
+    pub collection_name: String,
+    pub name: String,
+    pub amount: BigDecimal,
+    pub table_handle: String,
```

Comment:
what is the usage of the table_handle?

Reply:
That's a good question, it'll be mostly for debugging. To be clear we don't have to expose all the fields in this table. I don't think that we should expose this field either.